.github/workflows/ci.yml

```diff
@@ -381,3 +381,22 @@ jobs:
           asset_path: ./paperless-ngx.tar.xz
           asset_name: paperless-ngx-${{ steps.get_version.outputs.version }}.tar.xz
           asset_content_type: application/x-xz
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Append Changelog to docs
+        id: append-Changelog
+        working-directory: docs
+        run: |
+          git status
+          git pull origin main
+          git checkout main
```

Comment:
I think this could combine with the above checkout:
```
uses: actions/checkout@v3
with:
    ref: main
```

Reply:
Yep good call, that appears to work well.